lib/locations/locations.go

```diff
@@ -285,14 +285,18 @@ func userHomeDir() string {
 }
 
 func GetTimestamped(key LocationEnum) string {
-	// We take the roundtrip via "${timestamp}" instead of passing the path
+	return getTimestampedAt(key, time.Now())
+}
+
+func getTimestampedAt(key LocationEnum, when time.Time) string {
+	// We take the roundtrip via "%{timestamp}" instead of passing the path
 	// directly through time.Format() to avoid issues when the path we are
 	// expanding contains numbers; otherwise for example
 	// /home/user2006/.../panic-20060102-150405.log would get both instances of
 	// 2006 replaced by 2015...
 	tpl := locations[key]
-	now := time.Now().Format("20060102-150405")
-	return strings.ReplaceAll(tpl, "${timestamp}", now)
+	timestamp := when.Format("20060102-150405")
```

Comment:
Would it make sense to apply the same new pattern as in #9157?

Frankly I can't make much sense of that comment, and it seems very much related to this funky Go paradigm.

Reply:
No, not really. It's a different format and whatnot.